### DISCUSSION TITLE
Update Conda package

### DISCUSSION THREAD

**Discussion by Zethson** [pyudev/pyudev#436] [Ideas]
Hi,

the latest version on Conda is still 0.21, however 0.22.0 has been released.

Please bump the Conda package.

Thanks.

**Comment by mulkieran** [comment]
@Zethson: What Conda package are you talking about? I don't think I have any control over this.

**Comment by Zethson** [comment]
https://anaconda.org/pdrops/pyudev

**Comment by Zethson** [comment]
If this is not under your control I would kindly request that you maintain a Conda package.

**Comment by mulkieran** [comment]
Please feel free to submit a PR for the necessary changes.

**Comment by Zethson** [comment]
There are none required.

https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs-skeleton.html

guides you through creating a Conda package. Since maintainers of the PyPI package aka you should also maintain the Conda package (to prevent confusion like this post) I would prefer not to create the Conda package myself.

**Comment by mulkieran** [comment]
Try communicating with pdrops instead, as they seem to be already maintaining the package for this project.

**Comment by mulkieran** [comment]
https://anaconda.org/pdrops

**Comment by Zethson** [comment]
If only I knew how to contact people on Anaconda.
